automatic/k9s/tools/chocolateyinstall.ps1

```diff
@@ -5,18 +5,18 @@ $toolsDir      = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
 
 $packageArgs = @{
   PackageName    = $packageName
-  FileFullPath64 = Get-Item $toolsDir\k9s_*_Windows_x86_64.tar.gz
+  FileFullPath64 = Get-Item $toolsDir\k9s_*Windows_x86_64.tar.gz
   Destination    = $toolsDir
 }
 
 Get-ChocolateyUnzip @packageArgs
 
 $packageArgs2 = @{
   PackageName    = $packageName
-  FileFullPath64 = Get-Item $toolsDir\k9s_*_Windows_x86_64.tar
+  FileFullPath64 = Get-Item $toolsDir\k9s_*Windows_x86_64.tar
   Destination    = $toolsDir
 }
 
 Get-ChocolateyUnzip @packageArgs2
 
-Remove-Item "$toolsDir\k9s_*_Windows_*.tar"
+Remove-Item "$toolsDir\k9s_*Windows_*.tar"
```

Comment:
Can you change this line to
```suggestion
Remove-Item "$toolsDir\k9s_*Windows_*.tar","$toolsDir\k9s_*Windows_*.tar.gz"`
```

Seems like we are forgetting to remove the `.tar.gz` archive.

Reply:
Yeah, thats a good catch. I believe I can change that line to `Remove-Item "$toolsDir\k9s_*Windows_*.tar*"`. That should catch both of them. What do you think?

Reply:
@SayakMukhopadhyay
Just suggesting a non-global way. You could try using `Get-Version` on the asset filename. If that returns a version then it will be the new filename, and the old filename if nothing.
Just a suggestion.

Reply:
Please see other comment, thank you.

Reply:
> Yeah, thats a good catch. I believe I can change that line to Remove-Item "$toolsDir\k9s_*Windows_*.tar*". That should catch both of them. What do you think?

Yeah, that should work.